### ISSUE TITLE
Ability to see all my work across projects

### ISSUE THREAD

**Issue by ajefts** [topcoder-platform/work-manager#1175]
Feature request:

Currently, it is difficult to track all of my active work across projects.

As a user, I want to see a list of all draft and active work across all my projects. I want to be able to navigate to a view in work manager that list the work items and then allows me to click into each work item. Basic sorting and filtering would ideally be available.

**Comment by ajefts** [comment]
@vikasrohit What do you think about this idea?

**Comment by vikasrohit** [comment]
Yep, we had [this](https://github.com/topcoder-platform/work-manager/issues/904) in our road map. But it never get kicked off because of other priority features in WM. We can plan for it in next pitch.
